README.md

```diff
@@ -9,10 +9,16 @@ The MPI Operator makes it easy to run allreduce-style distributed training on Ku
 
 You can deploy the operator with default settings by running the following commands:
 
+- Latest Development Version
+
 ```shell
-git clone https://github.com/kubeflow/mpi-operator
-cd mpi-operator
-kubectl apply -f deploy/v2beta1/mpi-operator.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeflow/mpi-operator/master/deploy/v2beta1/mpi-operator.yaml
```

Comment:
this file still says latest actually https://github.com/kubeflow/mpi-operator/blob/382da780a7a9a23e6c9fd9dc8c93ba77aba81c48/deploy/v2beta1/mpi-operator.yaml#L483

regenerate the file

Reply:
I plan to generate that manifest in #509.
Do you prefer to regenerate the manifests in this PR?

Reply:
yes please

Reply:
Sure. I close #508 and regenerate manifests in this PR.